webapp/advantage/ua_contracts/api.py

```diff
@@ -109,6 +109,15 @@ def get_contract(self, contract_id: str):
 
         return contract
 
+    def get_account_offers(self, account_id: str):
```

Comment:
You can write unit tests for the endpoint.

You can see examples at: `/tests/advantage/test_api.py`. I think all our other api functions are covered.